.github/actions/build/action.yaml

```diff
@@ -33,30 +36,41 @@ runs:
           cd src
           cobalt/build/gn.py -p ${{ matrix.platform }} -C ${{ matrix.config }} --no-check
         shell: bash
-      - name: gn dump args
+      - name: Dump GN args
         run: |
           cd src
           gn args --list --short --overrides-only out/${{ matrix.platform }}_${{ matrix.config }}
         shell: bash
-      - name: ninja build
+      - name: Ninja build
         env:
           TARGETS: ${{ inputs.targets }}
         run: |
           set -x
           cd src
           ninja -C out/${{ matrix.platform }}_${{ matrix.config }} $(echo "${TARGETS}" | tr -d '"')
         shell: bash
-      - name: sccache check
+      - name: Dump sccache stats
         run: sccache -s
         shell: bash
-      - name: find apks
-        run: find src/out -name \*.apk -print
-        shell: bash
       - name: Archive Android APKs
-        if: startsWith(${{matrix.platform}}, 'android')
+        if: startsWith(${{matrix.platform}}, 'android') && matrix.config == 'qa'
```

Comment:
@oxve Is this change necessary?

Reply:
I don't remember why it is there. Probably not.